data/deep missions.txt

```diff
@@ -40,24 +40,29 @@ mission "Deep: Syndicate Convoy"
 	to offer
 		"mystery cube" >= 10
 		or
-			random < 30
+			random < 20
 			and
 				"deep convoy" == 1
-				random < 40
+				random < 30
 			and
 				"deep convoy" == 2
-				random < 50
+				random < 40
 		
 	on offer
 		conversation
-			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have DS ships do it," he says, "but we're short on available ships given certain circumstances."`
+			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have our own fleet do it," he says, "but we're short on available ships given certain circumstances."`
```

Comment:
The officer actually saying "DS ships" was intentionally casual, but I'll change it for the sake of not having people complain.